Dockerfile

```diff
@@ -67,10 +67,11 @@ RUN composer install $COMPOSER_FLAGS --no-scripts --no-autoloader
 
 COPY . .
 RUN composer install $COMPOSER_FLAGS \
-    && chown -R "${APP_USER_NAME}:${APP_USER_NAME}" var/
+ && chown -R "${APP_USER_NAME}:${APP_USER_NAME}" var/ \
+ && chown "${APP_USER_NAME}:${APP_USER_NAME}" vendor/infection/extension-installer/src/GeneratedExtensionsConfig.php
```

Comment:
Tohle je potreba kvuli `composer run post-install-cmd`. Teoreticky by se to dalo z entrypointu vyhodit (do dnes to nebylo potreba), pokud bysme se rozhodli tu entrypoint udelat optimalizovany na miru (jiny nez u appek).